packages/components/pie-textarea/src/index.ts

```diff
@@ -43,21 +70,89 @@ export class PieTextarea extends RtlMixin(LitElement) implements TextareaProps {
         }
     }, 100);
 
+    /**
+     * (Read-only) returns a ValidityState with the validity states that this element is in.
+     * https://developer.mozilla.org/en-US/docs/Web/API/HTMLObjectElement/validity
+     */
+    public get validity (): ValidityState {
+        return (this._textarea as HTMLTextAreaElement).validity;
+    }
+
+    /**
+     * Called after the disabled state of the element changes,
+     * either because the disabled attribute of this element was added or removed;
+     * or because the disabled state changed on a <fieldset> that's an ancestor of this element.
+     * @param disabled - The latest disabled state of the input.
+     */
+    public formDisabledCallback (disabled: boolean): void {
+        this.disabled = disabled;
+    }
+
+    /**
+     * Called when the form that owns this component is reset.
+     * Resets the value to the default value.
+     */
+    public formResetCallback (): void {
+        this.value = this.defaultValue ?? defaultProps.value;
+
+        // This ensures the input value is updated when the form is reset.
+        // Otherwise, there is a bug where values like 'e1212' for number inputs do not correctly reset.
+        if (this._textarea) {
+            this._textarea.value = this.value;
+        }
+
+        this._internals.setFormValue(this.value);
+    }
+
+    protected firstUpdated (_changedProperties: PropertyValues<this>): void {
+        super.firstUpdated(_changedProperties);
```

Comment:
It's [not necessary](https://lit.dev/docs/components/lifecycle/#firstupdated) to call `super` inside `firstUpdated`